pandas/tests/io/excel/test_readers.py

```diff
@@ -1185,6 +1185,13 @@ def test_no_header_with_list_index_col(self, read_ext):
         )
         tm.assert_frame_equal(expected, result)
 
+    def test_one_col_noskip_blank_line(self, read_ext):
+        file_name = "one_col_blank_line" + read_ext
```

Comment:
Add the issue number as the first line - e.g. `# GH 39808`

Reply:
done